firmware/libs/serialization/Makefile.dep

```diff
@@ -1 +1,3 @@
 USEPKG += tinycbor
+USEMODULE += slipdev
+USEMODULE += slipdev_l2addr
```

Comment:
new line at EOF

Reply:
this not must be here in serialization, was already changed.